src/quick-lint-js/parse-expression-inl.h

```diff
@@ -1149,6 +1149,21 @@ expression* parser::parse_expression_remainder(Visitor& v, expression* ast,
 
   // f(x, y, z)  // Function call.
   case token_type::left_paren:
+    switch (binary_builder.last_expression()->kind()) {
+    case expression_kind::arrow_function: {
```

Comment:
Nit: I think an `if` is better than a `switch`:
```suggestion
    if (binary_builder.last_expression()->kind() == expression_kind::arrow_function) {
```